tests/appsec/test_asm_standalone.py

```diff
@@ -56,6 +59,40 @@ def _assert_tags_value(span, obj, expected_tags):
         except (KeyError, AssertionError) as e:
             return False
 
+    def setup_check_product_is_enabled(func):
+        def wrapper(self):
+            headers = {}
+            if self.tested_product is "appsec":
```

Comment:
string comparison uses '==' : 

```suggestion
            if self.tested_product == "appsec":
```